src/utilities/include/utils.h

```diff
@@ -26,6 +26,15 @@
 #include "par.h"
 #include "vec.h"
 
+#if __has_include(<tracy/Tracy.hpp>)
+#include <tracy/Tracy.hpp>
+#else
+#define FrameMarkStart(x)
+#define FrameMarkEnd(x)
+#define ZoneScoped
+#define ZoneScopedN(name)
```

Comment:
Maybe a good place to document how we should use `ZoneScoped`? I have a vague idea from your usage, but I'm not entirely sure what it does.

Reply:
Sure, it is just something to mark the current zone (scope) for the profiler to see.